packages/forma-36-react-components/src/components/Card/EntryCard/EntryCard.tsx

```diff
@@ -207,15 +206,12 @@ export function EntryCard({
                 >
                   {contentType}
                 </div>
-                {statusIcon && typeof statusIcon === 'string' ? (
-                  <Icon
-                    icon={statusIcon as IconType}
-                    color="muted"
+                {StatusIcon ? (
+                  <StatusIcon
                     className={styles['EntryCard__icon']}
+                    variant="muted"
                   />
-                ) : (
-                  statusIcon
-                )}
+                ) : null}
```

Comment:
Is there a reason to return null? 

I think you could do this without returning null when StatusIcon is undefined, no?

```tsx
{StatusIcon && (
  <StatusIcon
    className={styles['EntryCard__icon']}
    variant="muted"
  />
)}
```

Reply:
No you're right, it's the same thing. I was trying an alternative at one point and needed the ternary and never changed it back, but it should be the same now 🙂